pyproject.toml

```diff
@@ -114,18 +116,38 @@ script = "scripts.test:test(dist=True, type_checker='pyright')"
 [tool.poe.tasks.stubtest]
 script = "scripts.test:stubtest(allowlist, check_missing, nightly)"
 help = "Run stubtest to compare the installed stubs against pandas"
-args = [{ name = "allowlist", positional = true, default = "", required = false, help= "Path to an allowlist (optional)" }, {name = "check_missing", positional = false, default = false, type = "boolean", required = false, help= "Report errors when the stubs are incomplete (off by default)"}, {name = "nightly", positional = false, default = false, type = "boolean", required = false, help= "Compare against pandas nightly (off by default)"}]
+args = [
+  { name = "allowlist", positional = true, default = "", required = false, help = "Path to an allowlist (optional)" },
+  { name = "check_missing", positional = false, default = false, type = "boolean", required = false, help = "Report errors when the stubs are incomplete (off by default)" },
+  { name = "nightly", positional = false, default = false, type = "boolean", required = false, help = "Compare against pandas nightly (off by default)" },
+]
 
 
 [tool.black]
 target-version = ['py39']
 
 [tool.isort]
 known_pre_libs = "pandas._config"
-known_pre_core = ["pandas._libs", "pandas._typing", "pandas.util._*", "pandas.compat", "pandas.errors"]
+known_pre_core = [
+  "pandas._libs",
+  "pandas._typing",
+  "pandas.util._*",
+  "pandas.compat",
+  "pandas.errors",
+]
 known_dtypes = "pandas.core.dtypes"
 known_post_core = ["pandas.tseries", "pandas.io", "pandas.plotting"]
-sections = ["FUTURE", "STDLIB", "THIRDPARTY" ,"PRE_LIBS" , "PRE_CORE", "DTYPES", "FIRSTPARTY", "POST_CORE", "LOCALFOLDER"]
+sections = [
+  "FUTURE",
+  "STDLIB",
+  "THIRDPARTY",
+  "PRE_LIBS",
+  "PRE_CORE",
+  "DTYPES",
+  "FIRSTPARTY",
+  "POST_CORE",
+  "LOCALFOLDER",
+]
```

Comment:
Do you use a formatting tool for these changes?

Reply:
I used VSCode to do the editing, and I have a setting to tell it to auto-format files after saving, so I'm guessing that's what happened.

Will you merge?